src/System.IO.FileSystem.Watcher/ref/System.IO.FileSystem.Watcher.cs

```diff
@@ -46,6 +46,14 @@ public partial class FileSystemWatcher
         public System.IO.WaitForChangedResult WaitForChanged(System.IO.WatcherChangeTypes changeType) { return default(System.IO.WaitForChangedResult); }
         public System.IO.WaitForChangedResult WaitForChanged(System.IO.WatcherChangeTypes changeType, int timeout) { return default(System.IO.WaitForChangedResult); }
     }
+    [Serializable]
```

Comment:
>Serializable [](start = 5, length = 12)

Just curious, should this type implement `ISerializable`? I ask because usually types that have this attribute also have that interface and it's implementation.